cheroot/test/test_conn.py

```diff
@@ -838,7 +838,8 @@ def test_Chunked_Encoding(test_client):
 
     # Try a chunked request that exceeds server.max_request_body_size.
     # Note that the delimiters and trailer are included.
-    body = b'3e3\r\n' + (b'x' * 995) + b'\r\n0\r\n\r\n'
+    body = '{}{}{}'.format('3e3\r\n', 'x' * 995, '\r\n0\r\n\r\n') \
+        .encode('ascii')
```

Comment:
Please use proper literals instead of heavy encode/decode methods. It's also a good place for `bytes.join()`. Same with escaping EOL.